setup.py

```diff
@@ -28,7 +28,7 @@
         # TODO: Investigate if it's needed. rotkehlchen.spec is where files seem to be copied
         'rotkehlchen': ['data/*.json', 'data/*.meta', 'data/*.db'],
     },
-    python_requires='>=3.6',
+    python_requires='>=3.9.6',
```

Comment:
Would we really require >=3.9.6 though?

Would nobody be able to run it with 3.9.1? OR 3.8? Gotta be careful here.